src/bokeh/models/glyphs.py

```diff
@@ -1576,6 +1581,77 @@ def __init__(self, *args, **kwargs) -> None:
 
     border_line_color = Override(default=None)
 
+@abstract
+class MathTextGlyph(Text):
+    """ Base class for math text glyphs. """
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+class MathMLGlyph(MathTextGlyph):
+    """ Render mathematical content using `MathML <https://www.w3.org/Math/>`_
+    notation.
+
+    See :ref:`ug_styling_mathtext` in the |user guide| for more information.
+
+    """
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+class TeXGlyph(MathTextGlyph):
+    """
+    Render mathematical content using `LaTeX <https://www.latex-project.org/>`_
+    notation.
+
+    See :ref:`ug_styling_mathtext` in the |user guide| for more information.
+
+    .. note::
+        Bokeh uses `MathJax <https://www.mathjax.org>`_ to render text
+        containing mathematical notation.
+
+        MathJax only supports math-mode macros (no text-mode macros). You
+        can see more about differences between standard TeX/LaTeX and MathJax
+        here: https://docs.mathjax.org/en/latest/input/tex/differences.html
+
+    """
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+    macros = Dict(String, Either(String, Tuple(String, Int)), help="""
+    User defined TeX macros.
+
+    This is a mapping from control sequence names (without leading backslash) to
+    either replacement strings or tuples of a replacement string and a number
+    of arguments.
+
+    Example:
+
+    .. code-block:: python
+
+        TeX(text=r"\\R \\rightarrow \\R^2", macros={"RR": r"{\\bf R}"})
+
+    """)
+
+    display = Either(Enum("inline", "block", "auto"), default="auto", help="""
+    Defines how the text is interpreted and what TeX display mode to use.
+
+    The following values are allowed:
+    * ``"auto"`` (the default)
+        The text is parsed, requiring TeX delimiters to enclose math content,
+        e.g. ``"$$x^2$$"`` or ``r"\\[\\frac{x}{y}\\]"``. This allows mixed
+        math text and regular text content. TeX display mode is inferred by
+        the parser.
+    * ``"block"``
+        The text is taken verbatim and TeX's block mode is used.
+    * ``"linline"``
+        The text is taken verbatim and TeX's inline mode is used.
+    """)
```

Comment:
```suggestion
    The following values are allowed:

    * ``"auto"`` (the default)
      The text is parsed, requiring TeX delimiters to enclose math content,
      e.g. ``"$$x^2$$"`` or ``r"\\[\\frac{x}{y}\\]"``. This allows mixed
      math text and regular text content. TeX display mode is inferred by
      the parser.
    * ``"block"``
      The text is taken verbatim and TeX's block mode is used.
    * ``"linline"``
      The text is taken verbatim and TeX's inline mode is used.
    """)
```

We have to add a blank line  and correct the typo in `"inline"`.

If you only add the blank line and don't change the indentation of the text, it will look like below.

![tex](https://github.com/bokeh/bokeh/assets/68053396/106686e8-1c98-4c0c-9e23-53e997a99ab2)

If you also change the indentation the outcome is:

![grafik](https://github.com/bokeh/bokeh/assets/68053396/ba08d0b7-ef51-4f2a-adc9-32da96c9fc87)

There are also 3 files to add in `bokeh/source/docs/reference/models/glyphs/`.
* `math_ml.rst`
* `math_text.rst`
* `tex.rst`

This is an example:

```
.. _bokeh.models.glyphs.MathTextGlyph:

MathTextGlyph
-------------

.. bokeh-model:: MathTextGlyph
    :module: bokeh.models.glyphs

```

@mattpap  I can add the files to this branch if you want, because I needed them to check the generated docs.

Reply:
> I can add the files to this branch if you want, because I needed them to check the generated docs.

@mosc9575, go ahead.